packages/coordinator/src/navigation/CoordinatorRouter.module.scss

```diff
@@ -1,3 +1,14 @@
 .content {
-  height: calc(100% - var(--height-header));
+  height: calc(100% - var(--height-header) - var(--height-footer));
+}
+
+$footer_height: var(--height-footer, 30px);
+
+.footer {
+  color: var(--color-mid-grey);
+  height: $footer_height;
+  position: absolute;
+  bottom: 0;
+  left: 2px;
+  font-size: 14px;
 }
```

Comment:
1. Since this div is positioned inside and with the flow, no reason to use absolute positioning (or any explicit position)
2. This part of the website is written in English only and can use the "direction: ltr" attribute to position it to the left

Reply:
3. Naturally, absolute positioning related attributes (bottom, left) can be replaced with relative attributes (padding, margin)